Commands.adoc

```diff
@@ -0,0 +1,226 @@
+= Available Commands and Examples
+:toc:
+:toc-placement!:
+
+Once you do `vagrant up` and run an ADB box, as a developer you may want to
```

Comment:
'do' seems a bit to generic to me. Either "once you execute" or just "Once you vagrant up", making 'vagrant up' basically a verb. I like the latter.